torch/hub.py

```diff
@@ -287,13 +352,25 @@ def list(github, force_reload=False, skip_validation=False):
             specified by the ``github`` argument properly belongs to the repo owner. This will make
             requests to the GitHub API; you can specify a non-default GitHub token by setting the
             ``GITHUB_TOKEN`` environment variable. Default is ``False``.
+        trust_repo (bool, string or None): ``"check"``, ``True``, ``False`` or ``None``.
```

Comment:
There are some indentation issues here and below that lead to incorrect rendered docs: https://docs-preview.pytorch.org/72060/hub.html

I think this should do the job:

```py
        trust_repo (bool, string or None): ``"check"``, ``True``, ``False`` or ``None``.
            This parameter helps ensuring that users only run code from repos that they trust.

            - If ``False``, a prompt will ask the user whether the repo should
              be trusted.
            - If ``True``, the repo will be added to the trusted list and loaded
              without requiring explicit confirmation.
            - If ``"check"``, the repo will be checked against the list of
              trusted repos in the cache. If it is not present in that list, the
              behaviour will fall back onto the ``trust_repo=False`` option.
            - If ``None``: this will raise a warning, inviting the user to set
              ``trust_repo`` to either ``False``, ``True`` or ``"check"``. This
              is only present for backward compatibility and will be removed in
              the future.

            Default is ``None`` and will eventually change to ``"check"`` in a future version.
```